docs/usage/automation.md

```diff
@@ -135,10 +161,25 @@ You can also add all observables and relationships generated by this component i
 
 Will filter out any entities in the current stage that do not match the filter conditions in the stage. If the original triggering entity is a container, such as a report or case, then the container itself cannot be filtered out, and will be passed to subsequent stages.
 
+## Enroll manually an entity into a playbook
+
+You can enroll individual entities in playbooks by using the action "Enroll in playbook" from the entity details view.
+
+![Enroll entity in playbook](assets/playbook_enroll_entity.png)
+
+This will open a drawer where you can choose the playbook you want to trigger on this entity.
+
+![Enroll entity in playbook](assets/playbook_enroll_select.png)
+
+In this list, you will find:
+
+* active playbooks that have set an "Available for manual enrollment / trigger" event source
+* active playbooks with "Listen knowledge events" event source which filters match the entity
```

Comment:
I need a validation of this part.

The filters set in the "Available for manual enrollment / trigger" event source are not used ?